specification/maps/data-plane/Render/preview/1.0/render.json

```diff
@@ -482,7 +482,7 @@
     "/map/statetile": {
       "get": {
         "description": "**Applies to**: S0 and S1 pricing tiers.\n\n\nFetches state tiles in vector format typically to be integrated into indoor maps module of map control or SDK. The map control will call this API after user turns on dynamic styling (see [Zoom Levels and Tile Grid](https://docs.microsoft.com/en-us/azure/location-based-services/zoom-levels-and-tile-grid))",
-        "operationId": "Render_GetMapStateTilePreview",
+        "operationId": "GetMapStateTilePreview",
```

Comment:
Please remove the "Preview" of this name, this has been live forever!

Reply:
Fixed